composer.json

```
@@ -0,0 +1,47 @@
+{
+    "name": "laravel-notification-channels/netgsm",
+    "description": "NetGSM notification channel for Laravel 5.5 & 6.x"
+    "homepage": "https://github.com/laravel-notification-channels/netgsm",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Talha Zekeriya Durmuş",
+            "email": "drtzack@gmail.com",
+            "homepage": "https://talhadurmus.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.3",
+        "illuminate/notifications": "~5.5 || ~6.0",
+        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0"
```

I'd make this the same as the `illuminate/notifications` dep
```suggestion
        "illuminate/support": "~5.5 || ~6.0"
```